executor/runtime/docker/docker.go

```diff
@@ -1811,7 +1819,7 @@ func setupNetworking(burst bool, c runtimeTypes.Container, cred ucred) (cleanupF
 	}
 	defer shouldClose(netnsFile)
 
-	setupCommand := exec.Command(vpcToolPath(), setupNetworkingArgs(burst, c)...) // nolint: gosec
+	setupCommand := exec.CommandContext(ctx, vpcToolPath(), setupNetworkingArgs(burst, c)...) // nolint: gosec
```

Comment:
Same deal, do we no longer have cleanup concerns like we did previously wrt context and command execution?

Reply:
"Different" concerns.